src/hooks/useFormElementOptions.ts

```diff
@@ -41,19 +45,44 @@ export default function useFormElementOptions<T>({
     })
   }, [conditionallyShownOptions, element.options, onFilter])
 
+  //
+  const withInjectedOptions = React.useMemo(() => {
+    return shownOptions.reduce<FormTypes.ChoiceElementOption[]>(
+      (memo, option) => {
+        //No replaceable stuff
+        if (
+          !formElementsService.ElementWYSIWYGRegex.test(option.label) &&
+          !formElementsService.ElementWYSIWYGRegex.test(option.value)
+        ) {
```

Comment:
I dont think we want to do this check here as it does not cater for user profile and task information, or anything else we add later to the replaceable logic.